package.json

```diff
@@ -2,7 +2,7 @@
   "name": "settings-sharelatex",
   "homepage": "www.sharelatex.com",
   "description": "A centralised settings system for ShareLaTeX",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
I never thought I'd be the guy who quibbles over semver numbers, but this change feels like a `1.1.0` :smile:

Reply:
Good catch, fixed in 75c812eecd6b6803de38a2f68f4d07edc63da064